From 37b3eccbe01bc8a01e21fa0548829a1c6d96ab22 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 11 Jan 2022 18:46:02 +0200
Subject: [PATCH] [SW-0] gaudi: preboot: increase image size

Latest code additions caused the FS build to exceed the 128kb that
was allocated for preboot.
Additional 20Kb were added to preboot image size

Change-Id: I2eb956f67814aa97e013b70a719fa3a0a56c519b
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi/arm/common.h b/arch/gaudi/arm/common.h
index 3c55d5c..369f4ac 100644
--- a/arch/gaudi/arm/common.h
+++ b/arch/gaudi/arm/common.h
@@ -10,7 +10,7 @@
 #include <address_space.h>
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x20000		/* 128KB */
+#define PREBOOT_SIZE			0x25000		/* 148KB */
 
 #define BOOT_INFO_BASE			HL_SP_SRAM_BASE
 
